static/src/js/actions/__tests__/errors.test.js

```diff
@@ -79,3 +84,36 @@ describe('removeError', () => {
     expect(removeError(payload)).toEqual(expectedAction)
   })
 })
+
+describe('handleError', () => {
+  test('should dispatch error with showAlertButton when showAlertButton parameter is true', async () => {
+    const consoleMock = jest.spyOn(console, 'error').mockImplementation(() => jest.fn())
+
+    nock(/localhost/)
+      .post(/error_logger/)
+      .reply(200)
+
+    const store = mockStore({})
+
+    await store.dispatch(handleError({
+      error: new Error('CMR error'),
+      action: 'fetchSubscriptions',
+      resource: 'subscription',
+      showAlertButton: true,
+      title: 'Custom error title'
+    }))
+
+    const storeActions = store.getActions()
+    expect(storeActions[0]).toEqual({
+      type: ADD_ERROR,
+      payload: expect.objectContaining({
+        title: 'Custom error title',
+        message: 'Error: CMR error',
+        showAlertButton: true,
+        notificationType: 'banner'
+      })
+    })
+
+    expect(consoleMock).toHaveBeenCalledTimes(1)
```

Comment:
Every time you use `toHaveBeenCalledTimes` you have to also use `toHaveBeenCalledWith`